.rspec

```diff
@@ -1,2 +1,2 @@
 --color
--rdebugger
+-rbyebug
```

Comment:
Can you remove this change? I'd rather discuss this separately in a different issue/PR.